examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h

```diff
@@ -38,6 +38,7 @@ class CHIPCommandBridge : public Command {
         : Command(commandName)
     {
         AddArgument("commissioner-name", &mCommissionerName);
+        AddArgument("commissioner-nodeId", 0, UINT64_MAX, &mCommissionerNodeId);
```

Comment:
This should probably have a docstring explaining the constraints on this (e.g. that it only works in non-interactive mode for now), right?

Reply:
I will add docstring. This does working in Interactive mode if pass in the arguments when starting interactive mode, but it only works for a single controller, because of how the stack starts with initializing the controllers.